standalone/ansible/roles/common/files/etc/ssh/sshd_config

```diff
@@ -25,7 +25,7 @@ LogLevel INFO
 
 # Authentication:
 LoginGraceTime 120
-PermitRootLogin without-password
+PermitRootLogin no
```

Comment:
So we first set up the root user, use it to create the `ubuntu` or `deploy` users, etc, and then disable root login?

Reply:
We manually create the `ubuntu` user, setup ssh for it and then run the scripts. Root ssh access is simply denied now.

Reply:
Noice.